CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,296 @@
+
+* Responsibilities:
+    * Must follow the [CNCF CoC](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)
+* How users can get involved with the community:
+    * Participating in community discussions
+    * Helping other users
+    * Submitting bug reports
+    * Commenting on issues
+    * Trying out new releases
+    * Attending community events
+
+
+### Contributor
+
+A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+* Responsibilities include:
+    * Follow the [CNCF CoC](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)
+    * Follow the project contributing guide
+* Requirements (one or several of the below):
+    * Report and sometimes resolve issues
+    * Occasionally submit PRs
+    * Contribute to the documentation
+    * Show up at meetings, takes notes
+    * Answer questions from other community members
+    * Submit feedback on issues and PRs
+    * Test releases and patches and submit reviews
+    * Run or helps run events
+    * Promote the project in public
+    * Help run the project infrastructure
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become an Organization Member
+
+
+### Organization Member
+
+An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges in both project repositories and elections, and as such are expected to act in the interests of the whole project.
+
+An Organization Member must meet the responsibilities and has the requirements of a Contributor, plus:
+
+* Responsibilities include:
+   * Continues to contribute regularly
+   * Help uphold our commmunity values and welcome newcomers
+* Requirements:
+   * Enabled [two-factor
+     authentication](https://help.github.com/articles/about-two-factor-authentication)
+     on their GitHub account
+   * Have made multiple contributions to the project or community. Contributions
+     may include, but is not limited to:
+      * Authoring or reviewing PRs on GitHub
+      * Filing or commenting on issues on GitHub
+      * Contributing to subprojects, or community discussions (e.g. meetings,
+        chat, email, and discussion forums)
+      * [Joined the Slack channel](https://cloud-native.slack.com/archives/C0344AANLA1)
+         * [Get an invite to join CNCF](http://slack.cncf.io/)
+      * Have read the [contributor
+        guide](https://github.com/open-feature/.github/blob/main/CONTRIBUTING.md)
+      * Actively contributing to 1 or more subprojects.
+* Privileges:
+    * May be assigned Issues and Reviews
+    * May give commands to CI/CD automation
```

Comment:
@beeme1mr is this actually true? If somebody is an org member, can they generally interact with CI/CD actions across all repos?